scousepy/stage_1.py

```diff
@@ -83,6 +85,16 @@ def calc_rms(spectrum):
 
     # Find all negative values
     negative_indices = (spectrum < 0.0)
+
+    if negative_indices.sum() < 2:
+        # spectrum is not continuum subtracted, maybe?
+        # first baseline-subtract the spectrum conservatively
+        # (note that this is a reassignment operation, it doesn't change the
+        # data)
```

Comment:
This could also go wrong if you have a really really tiny spectrum (like, <10 pixels), but that's not really worth SCOUSing in the first place...